roles/etcd/tasks/install.yml

```diff
@@ -17,6 +17,11 @@
            /usr/bin/docker cp etcdctl-binarycopy:{{ etcd_container_bin_dir }}etcdctl {{ bin_dir }}/etcdctl &&
            /usr/bin/docker rm -f etcdctl-binarycopy"
   when: etcd_deployment_type == "docker"
+  register: etcd_task_result
+  until: etcd_task_result.rc == 0
+  retries: 4
+  delay: "{{ 20 | random + 3 }}"
```

Comment:
it is pseudo random for each node and each retry iteration. It is from 3 to 23 x 4 max retries